In [2]:
import pandas

In [11]:
adopce = pandas.read_csv("adopce.csv", sep=";")
print(adopce)

       id             nazev_cz                     nazev_en       trida_cz  \
0       7          Šváb syčivý  Madagascar giant cockroach      Bezobratlí   
1       8  Pakobylka okřídlená     Pink-winged stick insect     Bezobratlí   
2      16                Tayra                        Tayra          Savci   
3      35        Kajmanka supí    Alligator snapping turtle          Plazi   
4      37      Želva žlutočelá       Indochinese box turtle          Plazi   
..    ...                  ...                          ...            ...   
508  1417  Rosnice šalamounská                               Obojživelníci   
509  1418       Létavka obecná                               Obojživelníci   
510  1419          Bažant paví                                       Ptáci   
511  1420  Sojkovec rezavouchý                                       Ptáci   
512  1421   Kachnice kaštanová                                       Ptáci   

     cena  k_prohlidce  
0    1000            1  
1    1000    

In [12]:
adopce.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 513 entries, 0 to 512
Data columns (total 6 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   id           513 non-null    int64 
 1   nazev_cz     513 non-null    object
 2   nazev_en     513 non-null    object
 3   trida_cz     513 non-null    object
 4   cena         513 non-null    int64 
 5   k_prohlidce  513 non-null    int64 
dtypes: int64(3), object(3)
memory usage: 24.2+ KB


In [13]:
print(adopce.shape)

(513, 6)


= počet řádků 513, počet sloupců 6
- na první místě počet je vždy počet řádků a na druhém počet sloupců

In [14]:
print(adopce.shape[0])

513


názvy sloupců:

In [15]:
print(adopce.columns)

Index(['id', 'nazev_cz', 'nazev_en', 'trida_cz', 'cena', 'k_prohlidce'], dtype='object')


In [16]:
print(adopce.iloc[34])

id                     99
nazev_cz        Ibis bílý
nazev_en       White ibis
trida_cz            Ptáci
cena                 3000
k_prohlidce             1
Name: 34, dtype: object


In [25]:
print(adopce.iloc[34, 1])

Ibis bílý


In [28]:
print(adopce.iloc[34, [1,2]])

nazev_cz     Ibis bílý
nazev_en    White ibis
Name: 34, dtype: object
